packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/SelectedItemWithMenu.tsx

```diff
@@ -33,7 +33,7 @@ export class SelectedItemWithMenu extends React.Component<IPeoplePickerItemWithM
         <div className='ms-PickerItem-content'>
           <Persona
             { ...item }
-            presence={ item.presence ? item.presence : PersonaPresence.online }
+            presence={ item.presence != null ? item.presence : PersonaPresence.none }
```

Comment:
FWIW i think passing null for item.presence is unexpected behavior because it's not a valid value for the enum. I think the right design in this case is, provide it or don't. I think typescript 2 can help with this.

https://blog.mariusschulz.com/2016/09/27/typescript-2-0-non-nullable-types